### ISSUE TITLE
 Duplicates shouldn't be allowed in Guest List?

### ISSUE THREAD

**Issue by mohitjagjit** [coloredcow/WP-RSVP#79]
<img width="1262" alt="screen shot 2017-09-19 at 12 14 57 pm" src="https://user-images.githubusercontent.com/16541548/30578993-4c8f28a0-9d34-11e7-94d5-6c31e019355c.png">

**Comment by wevekamola** [comment]
Yes, Mohit You are right Duplicates shouldn't be allowed. We can make sure that the same Email id can't be stored in the database.
